frontend/app/services/MemberService.scala

```diff
@@ -585,7 +587,8 @@ class MemberService(identityService: IdentityService,
                                         payPalBaid: String): Future[UpdateResult] =
     for {
       sub <- subscriptionService.current[SubscriptionPlan.Member](contact).map(_.head)
-      result <- zuoraService.createPayPalPaymentMethod(sub.accountId, payPalBaid, contact.email.getOrElse(""))
+      maybeEmail <- zuoraRestService.getAccount(sub.accountId) map (_.toOption.flatMap(_.billToContact.email))
```

Comment:
it's a shame we have to make an extra call here now, I wonder what `subscriptionService.current` does that we can't get from zuora.  Now that we have the identity id in zuora it might be doable @jacobwinch